JoonasHeinonen/escape-from-zordoom-prison
# Feature/in game UI

### [Improve player UI](https://trello.com/c/dLF52bZL/58-improve-player-ui-in-game)

![image](https://user-images.githubusercontent.com/33086486/215213932-f02d6a4a-eae5-4b13-832d-8edf1252e587.png)

**Changes:**
 - Created the initial layout for the in-game UI.
 - Created the graphical elements for the UI.
 - Created a theme for the in-game UI.
 - Created an additional script to hide the UI.

**How to test:**
 - See that the menu looks good and is according to the R&C: Rift Apart menu layout.
 - See that the in-game UI is hidden when you're either on vendor, inventory or in-game menu UI.
